app.js

```diff
@@ -81,30 +151,58 @@ function clearUI() {
     dinamicContent.innerHTML = '';
 }
 
-function displayDialog(repo) {
+async function displayDialog(repo) {
     dialogContent.innerHTML = `
-        <h2 class="text-center"> ${repo.name} </h2>
-        <div class="dialog__repo-card flex flex-wrap justify-around">
-            <p class="m-0"> 
-            <span class="text-bold"> Owner: </span> ${repo.owner.login} </p>
-            ${repo.language !== null ? (`
+            <h2 class="text-center" > ${repo.name} </h2>
+                <div class="dialog__repo-card flex flex-wrap justify-around">
+                    <p class="m-0">
+                        <span class="text-bold"> Owner: </span> ${repo.owner.login} </p>
+                    ${repo.language !== null ? (`
```

Comment:
It seems like nothing changed here, just indentation right? Let's revert these changes, so we see no differences here.